components/themeSelector.html

```diff
@@ -1,150 +0,0 @@
-<!-- FIXME: Please remove this weird script below -->
```

Comment:
How should we replace the theme selectors in Sitetheory for the new selectors you've built?

Reply:
@alexgurrola the new admin theme selector will contain the old select + new logic so I think we don't need to separate it. Maybe the naming of selector is confusing you so just tell me if you want to change to another name

Reply:
That's understandable.  I'd be happy to have one selector, and will be testing this out today.